src/skins-qt/menus.cc

```diff
@@ -90,7 +90,7 @@ static const audqt::MenuItem main_items[] = {
 
 static const audqt::MenuItem playback_items[] = {
     audqt::MenuCommand ({N_("Song Info ..."), "dialog-information", "I"}, audqt::infowin_show_current),
-    audqt::MenuCommand ({N_("Playback History ..."), "view-history", "H"}, action_playback_history),
+    audqt::MenuCommand ({N_("Playback History ..."), nullptr, "H"}, action_playback_history),
```

Comment:
`locate view-history` finds the `view-history` icon in 3 icon themes on my system: _breeze_, _breeze-dark_ and _oxygen_. What happens when the icon is missing, as I assume is the case on your system?

Reply:
Then this icon seems to be KDE specific. When it is missing no icon is shown.

Reply:
Fair enough, please check my two last commits from my [fork](https://github.com/radioactiveman/audacious-plugins/commits/master/). Can I merge them into the main repo?

Reply:
Both commits look good, thank you. Please merge.

By the way, defining [`QT_NO_CAST_FROM_ASCII`](https://doc-snapshots.qt.io/qt6-dev/qstring.html#QT_NO_CAST_FROM_ASCII) project-wide is considered a good Qt practice. KDE projects define it [by default](https://api.kde.org/ecm/kde-module/KDECompilerSettings.html#modern-code).